prowler/providers/aws/services/acm/acm_certificates_insecure_algorithms/acm_certificates_insecure_algorithms.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "acm_certificates_insecure_algorithms",
+  "CheckTitle": "Check if ACM Certificates use a secure RSA key size",
+  "CheckType": [
+    "Data Protection"
+  ],
+  "ServiceName": "acm",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:acm:region:account-id:certificate/resource-id",
+  "Severity": "high",
+  "ResourceType": "AwsCertificateManagerCertificate",
+  "Description": "Check if ACM Certificates use a secure RSA key size",
```

Comment:
```suggestion
  "Description": "Check if ACM Certificates use a secure RSA key length",
```

Reply:
Can you elaborate it more? Mention for example what is considered to be a secure key algorithm :)